content/docs/fairing/gcp-ai-platform-notebook.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Train and Deploy on GCP from an AI Platform Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform (GCP) from a notebook that is hosted on Google Cloud AI Platform"
+weight = 35
++++
+
+This guide introduces you to using [Kubeflow Fairing][fairing-repo] to train and
```

Comment:
Now that we have an overview of Fairing, I'd recommend linking to the overview rather than the repo here. The overview can link to the repo.

Reply:
Good point, done